### PR TITLE
Do not break text elements across pages when exporting reports to PDF

### LINKED ISSUE #3166: pdf report conversion sometimes cuts text lines across pages
Screenshot 2022-02-15 at 10 53 44

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #3166

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. On cg-vm1 (scout-stage) note that the PDF report of [this case](https://scout-stage.scilifelab.se/cust003/15001) has the problem
2. Clear browsing data
3. Switch to this branch and the exported should look better (no lines of text broken across lines)

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
Weird that on my local instance (not using the Docker image) the text in the case report is not broken. Could be Docker-related?

**Comment by northwestwitch** [comment]
Master branch:

image

<hr>

This branch: 🆗 

image

**Review by dnil — APPROVED** [review]
Nice! What about the `dismissed_panel`- looked like it was still susceptible on a quick read?

**Comment by northwestwitch** [comment]
> Nice! What about the `dismissed_panel`- looked like it was still susceptible on a quick read?

Thanks, fixed!

File: scout/server/blueprints/cases/templates/cases/case_report.html [hunk 1]
```diff
@@ -1269,7 +1269,7 @@ <h6 class="card-subtitle mb-2 text-muted"><span class="badge badge-warning">Part
             {% if variant.chromosome == variant.end_chrom %}
               chr{{variant.chromosome}}:{{variant.position}}-{{variant.end}}
             {% else %}
-              chr{{variant.chromosome}}:{{variant.position}}/{{'chr'+variant.end_chrom}}:{{variant.end}}
+              chr{{variant.chromosome}}:{{variant.position}}/{{'chr'+variant.end_chrom or variant.chromosome}}:{{variant.end}}
```
Comment (dnil) [inline scout/server/blueprints/cases/templates/cases/case_report.html:1272]: 👍

**Comment by northwestwitch** [comment]
Oh come on! The more you fix the code on this report and the more "bugs" you get: 😫 

image

**Comment by dnil** [comment]
Yeah, that's a bit odd! I remember seeing a lot on the html when checking through the "bugs" on main, but surprising it discovers it gradually! Maybe it doesn't add everything in the file even, only stuff that is more closely touched by the PR?

**Comment by northwestwitch** [comment]
> Yeah, that's a bit odd! I remember seeing a lot on the html when checking through the "bugs" on main, but surprising it discovers it gradually! Maybe it doesn't add everything in the file even, only stuff that is more closely touched by the PR?

I think it's because I touched those lines, then it checks them and finds stuff..

**Pull request merged**